performance/src/main/kotlin/org/jetbrains/ring/launcher.kt

```diff
@@ -96,6 +102,7 @@ class Launcher(val numWarmIterations: Int, val numberOfAttempts: Int) {
         runOctoTest()
 
         printResultsNormalized()
```

Comment:
Maybe move `printResultsNormalized` from `Launcher` just like you moved JSON dumping before? And implement following logic: if user provides `jsonReport` in `main` we write results to file and to stdout otherwise. 
Also it makes sense to remove `Results` in favor for `BenchmarkResult`.

Reply:
I stay current flow working as is until all analysis isn't placed in separate project. When it's ready I'll remove extra logic from benchmark application.